site/layouts/partials/sidebar-tree.html

```diff
@@ -68,7 +68,7 @@
   <ul class="td-sidebar-nav__section pr-md-3">
     <li class="td-sidebar-nav__section-title">
       <!-- Link for subections -->
-      <a href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $expandSection }} collapsed{{ end }}{{ if $activeSection}} active{{ end }} td-sidebar-link td-sidebar-link__section">{{ $s.LinkTitle }}</a>
+      <a href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $expandSection }} collapsed{{ end }}{{ if $activeSection}} active{{ end }} td-sidebar-link td-sidebar-link__section">🍀 {{ $s.LinkTitle }}</a>
```

Comment:
how about using this character  ▀ instead ?

Reply:
used &#11035; (solid black square )